src/corona_hakab_model/sparse_matrix/sparse.py

```diff
@@ -23,6 +23,9 @@
 else:
     import _sparse
 
+# todo remove this and add generator to cpp build
+from consts import generator
```

Comment:
add this line to build.py (even if you can't run it)

Reply:
done.
for now kept it both here and added it to build.py